.github/workflows/CD-adot-operator.yml

```diff
@@ -0,0 +1,49 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+name: Mirror ADOT Operator
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'the desired version number of the operator (ex. 0.45.0)'
+        required: true
+
+jobs:           
+  release-latest-image:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.RELEASE_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.RELEASE_SECRET }}
+          aws-region: us-west-2
+
+      - name: Version checking
+        id: version
+        run: VERSION="${{ github.event.inputs.version }}" bash tools/workflow/version-check.sh
```

Comment:
Is this collector or operator version? Does this need to go into the CD workflow?

Reply:
This is the Operator version, and I don't think it needs to go in the CD workflow since it should only run on release and also checks collector version; also this should be `tools/release/adot-operator-images-mirror/version-check.sh`, fixed that just now.